README.md

```diff
@@ -1,40 +1,23 @@
 # pisces
 
-
 <!-- WARNING: THIS FILE WAS AUTOGENERATED! DO NOT EDIT! -->
```

Comment:
^^ Just cause I'm curious, is it still the case that the file is autogenerated?

Reply:
It is! generated using `nbdev_docs` from `nbs/index.ipynb`